Resources/config/services.xml

```diff
@@ -53,6 +53,7 @@
         <service id="Trikoder\Bundle\OAuth2Bundle\Security\Authentication\Provider\OAuth2Provider">
             <argument key="$userProvider" />
             <argument key="$resourceServer" type="service" id="League\OAuth2\Server\ResourceServer" />
+            <argument key="$oauth2TokenFactory" type="service" id="Trikoder\Bundle\OAuth2Bundle\Security\Authentication\Token\OAuth2TokenFactory"
```

Comment:
@rjwebdev You forgot to self-close the tag, whoops!

Reply:
Closed the tag